tests/extension/oneapi_current_device/CMakeLists.txt

```
@@ -0,0 +1,5 @@
+if(SYCL_CTS_ENABLE_EXT_ONEAPI_CURRENT_DEVICE_TESTS)
+    file(GLOB test_cases_list *.cpp)
```

nit:

```suggestion
    set(test_cases_list
      current_device.cpp
    )
```

(to be more precise)

---

Looking at other CMakeLists, this approach 
`file(GLOB test_cases_list *.cpp)`
is standard

---

It is not like standard or non-standard. We have the only one source file in the folder, and it is more precise to mention only this file in CMakeLists.txt. Anyway, I marked this as "nit", so it is optional